context/app/static/js/components/workspaces/WorkspacesList.jsx

```diff
@@ -8,10 +8,10 @@ import { DeleteIcon, AddIcon } from 'js/shared-styles/icons';
 import { SpacedSectionButtonRow } from 'js/shared-styles/sections/SectionButtonRow';
 import { PanelWrapper } from 'js/shared-styles/panels';
 
+import WorkspaceDetails from 'js/components/workspaces/WorkspaceDetails';
```

Comment:
```suggestion
import WorkspaceDetails from './WorkspaceDetails';
```

When the IDE fills in imports, it doesn't take account of the ones that could be relative. Is this worth fixing by hand?